app/internal/load_jokes.py

```diff
@@ -0,0 +1,38 @@
+import json
+from typing import Dict, List, Optional
+
+from app.database.models import Joke
+
+from sqlalchemy.orm import Session
+
+
+def get_jokes_from_json() -> List[Dict[str, Optional[str]]]:
+    """Reading all jokes from the a JSON file that have been """\
+        """copied from 'http://api.icndb.com/jokes'"""
+    try:
+        with open('app/resources/jokes.json', 'r') as f:
+            jokes_list = json.load(f)
+    except (IOError, ValueError):
+        return []
+    return jokes_list
+
+
+def add_jokes_to_db(session: Session) -> None:
+    """This function reads the jokes and inserts them into the db"""
+    all_jokes = get_jokes_from_json()
+    jokes_objects = [
+        Joke(text=joke['text'])
+        for joke in all_jokes
+        ]
```

Comment:
This seems to be the same code as json_data_loader.py, why not use it instead?

Reply:
I changed that